source/org/zfin/sequence/reno/repository/HibernateRenoRepository.java

```diff
@@ -263,10 +306,7 @@ public List<RunCandidate> getSortedNonZFRunCandidates(Run run, String comparator
 
 
     public RunCandidate getRunCandidateByID(String runCandidateID) {
-        Session session = HibernateUtil.currentSession();
-        Criteria criteria = session.createCriteria(RunCandidate.class);
-        criteria.add(Restrictions.eq("zdbID", runCandidateID));
```

Comment:
Is this method not used?

Reply:
This method is used in a few places.  Why do you ask?

Reply:
never mind.